lib/modules/userTagger.js

```diff
@@ -80,6 +104,36 @@ module.options = {
 		advanced: true,
 		dependsOn: options => options.trackVoteWeight.value,
 	},
+	showPresetTags: {
```

Comment:
If all this does is determining whether the "presets" row is visible, it likely is not necessary. Just not generating the row should suffice.

Reply:
Is there a way to conditionally include the presets row in the dialog? I tried something like this within the `body` assignment but it wouldn't load the dialog even though the HTML immediately after assignment was completely fine. 

```
${presetTags.length > 0 && string._html`
    <div class="fieldPair">
        <label class="fieldPair-label" for="userTaggerPresetTags">Presets</label>
        <span id="userTaggerPresetTags"></span>
    </div>`
}
```